Add images section to troubleshooting in MSR (#25010)
DataDog/documentation: content/en/real_user_monitoring/session_replay/mobile/troubleshooting.md

- Views that use direct canvas drawing
- Advanced text styling

### Images do not render properly

> Depending on the SDK configuration of Mobile Session Replay images may not always be displayed. 

Reviewer: ```suggestion
Depending on the SDK configuration of Mobile Session Replay, images may not always be displayed. 
```

> If privacy settings allow for it, Mobile Session Replay will only capture Android images up to 100x100dp and iOS bundled images. All other types of images will be replced by a placeholder indicating its size in the UI.

Reviewer: ```suggestion
If privacy settings allow for it, Mobile Session Replay only captures Android images up to 100x100dp and iOS bundled images. All other images are replaced by a placeholder indicating the image size in the UI.
```

> Images follow a separate processing pipeline which may introduce a small delay between a replay being uploaded and an image being available for rendering in the web player. 

Reviewer: ```suggestion
Images follow a separate processing pipeline, which may introduce a small delay between uploading a replay and the image being available for rendering in the web player.
```

> In this case you should wait a few minutes and reload the replay. 

Reviewer: ```suggestion
In this case, wait a few minutes and then reload the replay. 
```
